### DISCUSSION TITLE
PacketSenderCLI.pro and Qt dependencies

### DISCUSSION THREAD

**Discussion by maksokami** [dannagle/PacketSender#281] [Q&A] [answered]
Hi,

I was just trying to compile PacketSenderCLI.pro and noticed that it still depends on Qt (and Qt on my edge device is way outdated and cannot be updated easily). Are there any plans to remove Qt from dependencies for CLI version?

Thank you

**Comment by dannagle** [comment, marked as answer]
There are no plans to remove Qt. That is too much effort, especially with [the new CLI-based intense traffic generator ](https://github.com/dannagle/PacketSender/tree/development#cliintensetraffic) in the development branch. 

Compiling the CLI version does remove some of the Qt GUI dependencies. 

Hope that helps,
Dan
